lib/osf-components/addon/services/osf-modal-state.ts

```diff
@@ -17,39 +17,33 @@ export default class OsfModalState extends Service {
             this.priorActiveElement = activeElement;
         }
 
-        // Update global state in another runloop to avoid "modified twice in one render" errors
-        run.next(this, function() {
```

Comment:
Why is this no longer necessary?

Reply:
the error mentioned in the comment was caused by the removed render modifier 
`did-insert`